src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,64 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
+import { SortFields } from '../../types/enum';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  searchValue: string;
+  filterField: SortFields;
+  onSearchValue: (v: string) => void;
+  onChangeFilterField: (f: SortFields) => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<Props> = ({
+  searchValue,
+  onSearchValue,
+  filterField,
+  onChangeFilterField,
+}) => {
+  const handleChangeFilter = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    onChangeFilterField(e.target.value as SortFields);
+  };
+
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            onChange={handleChangeFilter}
+            value={filterField}
+            data-cy="statusSelect"
+          >
+            <option value={SortFields.All}>{SortFields.All}</option>
+            <option value={SortFields.Active}>{SortFields.Active}</option>
+            <option value={SortFields.Comleted}>{SortFields.Comleted}</option>
```

Comment:
You can get values from enum using `Object.values` so you can map throw them and create options